extensions/BugModal/web/bug_modal.js

```diff
@@ -1011,8 +1011,6 @@ $(function() {
     $('#product, #component')
         .change(function(event) {
             $('.set-default-container').show();
-            $('#set-default-assignee').prop('checked', $('#assigned_to').val() == BUGZILLA.default_assignee).change();
-            $('#set-default-qa-contact').prop('checked', $('#qa_contact').val() == BUGZILLA.default_qa_contact).change();
             slide_module($('#module-people'), 'show');
```

Comment:
so where did these get moved to? we can't just delete them from bug_modal.js can we?